doc/nvim-tree-lua.txt

```diff
@@ -1446,10 +1446,26 @@ api.tree.toggle_help()                       *nvim-tree.api.tree.toggle_help()*
     - navigate.sibling.last
     - navigate.parent
     - navigate.parent_close
-    - navigate.git.next
-    - navigate.git.prev
-    - navigate.diagnostics.next
-    - navigate.diagnostics.prev
+
+api.node.navigate.git.next()               *nvim-tree.api.navigate.git.next()*
+    Navigate to the next item showing git status.
+
+api.node.navigate.git.prev()               *nvim-tree.api.navigate.git.prev()*
+    Navigate to the previous item showing git status.
+
+                                   *nvim-tree.api.navigate.diagnostics.next()*
+api.node.navigate.diagnostics.next() 
+    Navigate to the next item showing diagnostic status.
+
+                                   *nvim-tree.api.navigate.diagnostics.prev()*
+api.node.navigate.diagnostics.prev()
+    Navigate to the next item showing diagnostic status.
+
+api.node.navigate.open.next()             *nvim-tree.api.navigate.open.next()*
```

Comment:
Not sure is `open` is a good name here. It sorta points toward action rather than a state, such as `diagnostics` or `git`. I don't have a better idea though. `opened`, `buffers` or more verbose like `opened_buffers`, `listed_buffers`. Not sure.

Reply:
> `listed_buffers`

That's factual and exactly what we are testing for. Updated.

Reply:
Still not right... changed to `opened` to match `highlight_opened_files`.

Clarified that both of these actually show `bufloaded()`.

Noted that `filters.no_buffer` uses `buflisted()`, however that could be considered a breaking change and may also have side effects RE buffer unloading.